### ISSUE TITLE
Standalone Vocal Splitter

### ISSUE THREAD

**Issue by EliteMasterEric** [gusborg88/fnf-porter#14]
### What category does this fall into?

File conversion

### Additional Context (if applicable)

I have a bunch of songs where I want to run the vocal splitter, but if they aren't specifically from a Psych mod folder, I have to set one up specifically to use the tool on it. It would be great if there was a separate pane (or maybe a separate tool if that makes more sense) to provide a chart file and an audio file and have FFMPEG do the work of splitting.

**Comment by BombasticTom** [comment]
Jank is working in it! 
![Screenshot_20240523_093204_edit_464285999047523](https://github.com/gusborg88/fnf-porter/assets/85838311/35aa970f-414e-4e5e-9a9a-3dfb772d2d71)

**Comment by tposejank** [comment]
Yes. seems Vocal Split was limited to this issue. We plan on arranging a new window design soon, maybe for the meantime Vocal Split may become an alternative tool (just itself, but still being here). Thanks for the suggestion!

**Comment by tposejank** [comment]
Being worked on in v1.0 branch.
